mod-source-record-storage-server/src/main/java/org/folio/services/handlers/match/MarcHoldingsMatchEventHandler.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.services.handlers.match;
+
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_SRS_MARC_HOLDINGS_RECORD_MATCHED;
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_SRS_MARC_HOLDINGS_RECORD_NOT_MATCHED;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.folio.dao.RecordDao;
+import org.folio.services.util.TypeConnection;
+
+public class MarcHoldingsMatchEventHandler extends AbstractMarcMatchEventHandler {
+
+  @Autowired
+  public MarcHoldingsMatchEventHandler(RecordDao recordDao) {
+    super(TypeConnection.MARC_HOLDINGS, recordDao, DI_SRS_MARC_HOLDINGS_RECORD_MATCHED,
```

Comment:
You should register event `DI_SRS_MARC_HOLDINGS_RECORD_MATCHED` in `DataImportConsumersVerticle`